### PR TITLE
explain LinkedPipes ETL configuration

### PR DESCRIPTION
**Pull request by costezki**

### CODE REVIEW

File: docker/.env [hunk 1]
```diff
@@ -24,13 +24,12 @@ RDF_VALIDATOR_GUNICORN_TIMEOUT=1200
...
 DIFFER_UI_URL=http://dev.meaningfy.ws:8030
-LINKEDPIPES_ETL_UI_URL=http://dev.meaningfy.ws:8060
 
 # LinkedPipesETL variables
 LP_ETL_USER=1983
 LP_ETL_BUILD_BRANCH=master
-LP_ETL_DOMAIN=http://dev.meaningfy.ws:8060
-#LP_ETL_DOMAIN=http://localhost:8060
+#LP_ETL_DOMAIN=http://dev.meaningfy.ws:8060
+LP_ETL_DOMAIN=http://localhost:8060
```
Comment (cosletmihai) [inline docker/.env:32]: which one should be left as active?

File: docs/tech-manual/content/configuration.tex [hunk 2]
```diff
@@ -220,18 +220,19 @@ \section{Configuration}
...
-		Executor monitor domain & http://localhost:8060 & LP\_ETL\_DOMAIN \\
+		Executor monitor service port & 8061 & LP\_ETL\_MONITOR\_PORT \\		
 		Service UI port & 8060 & LP\_ETL\_PORT \\
-		UI domain & http://localhost:8060 & LP\_ETL\_DOMAIN \\* \bottomrule
+		Service domain & http://localhost:8060 & LP\_ETL\_DOMAIN \\* \bottomrule
 		\caption{LinkedPipes ETL services configurations}
 		\label{tab:my-table5}\\
 	\end{longtable}
 	
+	Note that it is important to change the LP\_ETL\_DOMAIN each time the deployment environment changes. If it runs locally please set the value to be \textit{http://localhost:8060}, otherwise if it runs on a dedicated domain, the variable value must be set accordingly. If this is not done, the services will run but will not be able to load any pipelines because of a mismatch between the domain in the pipeline URI (created by using the variable value) and the domain of the service host.
```
Comment (cosletmihai) [inline docs/tech-manual/content/configuration.tex:232]: you can use `\texttt{LP\_ETL\_DOMAIN}` to use monospace text and signify that this is a env variable.

File: docs/tech-manual/content/configuration.tex [hunk 3]
```diff
@@ -220,18 +220,19 @@ \section{Configuration}
...
 		\label{tab:my-table5}\\
 	\end{longtable}
 	
+	Note that it is important to change the LP\_ETL\_DOMAIN each time the deployment environment changes. If it runs locally please set the value to be \textit{http://localhost:8060}, otherwise if it runs on a dedicated domain, the variable value must be set accordingly. If this is not done, the services will run but will not be able to load any pipelines because of a mismatch between the domain in the pipeline URI (created by using the variable value) and the domain of the service host. 
+	
+	The special \textit{configuration.properties} file, as mentioned above must be in synch with the port numbers of the environment variables. The meaning of these variables is explained on the LinkedPipes ETL website. They have been preconfigured, so that no changes are necessary there but of course they can be adjusted if needed. The native configuration file is available next to the \textit{docker-compose.yml} in the subfolder \textit{linkedpipes-etl/configuration}. The content of this file with descriptions of parameters as provided by the LinkedPipes ETL authors is available in the Section \ref{sec:appendinx1}.
```
Comment (cosletmihai) [inline docs/tech-manual/content/configuration.tex:234]: I'd use monospace font for `configuration.properties`

**Review by cosletmihai — APPROVED** [review]

File: docker/.env (hunk 1 above)
Comment (costezki) [reply docker/.env:32]: localhost

File: docs/tech-manual/content/configuration.tex (hunk 2 above)
Comment (costezki) [reply docs/tech-manual/content/configuration.tex:232]: Italics have been used so far for this purpose

File: docker/.env (hunk 1 above)
Comment (costezki) [reply docker/.env:32]: oh, no! dev.meaningfy for our testing

**Pull request merged**